pom.xml

```diff
@@ -23,17 +28,22 @@
         <dependency>
             <groupId>org.bukkit</groupId>
             <artifactId>bukkit</artifactId>
-            <version>1.7.2-R0.3</version>
+            <version>1.14.4-R0.1-SNAPSHOT</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>com.acrobot.chestshop</groupId>
             <artifactId>chestshop</artifactId>
-            <version>3.7.5</version>
+            <version>3.9.2</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/chestshop-3.9.2.jar</systemPath>
         </dependency>
         <dependency>
             <groupId>com.palmergames.bukkit</groupId>
             <artifactId>towny</artifactId>
-            <version>0.84.0.0</version>
+            <version>0.94.0.6</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/towny-0.94.0.6.jar</systemPath>
```

Comment:
This will stop any way of building this project automatically as the file is missing (also the systemPath property is deprecated). Dependency artifacts should either be directly gotten via their maven repository or be installed into the included one with the `install_dependency_to_repo.sh` script if they for some reason don't provide a repo on their own. (Which I assume is the case here?)